test/test_testing.py

```diff
@@ -1180,30 +1194,6 @@ def test_mapping_mismatching_values_msg(self):
             torch.testing.assert_close(actual, expected)
 
 
-class TestAssertCloseComplex(TestCase):
-    def test_mismatching_nan_with_equal_nan(self):
-        actual = torch.tensor(complex(1, float("NaN")))
-        expected = torch.tensor(complex(float("NaN"), 1))
-
-        for fn in assert_close_with_inputs(actual, expected):
-            with self.assertRaises(AssertionError):
-                fn(equal_nan=True)
-
-    def test_mismatching_nan_with_equal_nan_relaxed(self):
-        actual = torch.tensor(complex(1, float("NaN")))
-        expected = torch.tensor(complex(float("NaN"), 1))
-
-        for fn in assert_close_with_inputs(actual, expected):
-            fn(equal_nan="relaxed")
-
-    def test_matching_conjugate_bit(self):
```

Comment:
Don't we still need this test?

Reply:
#56058 removed the parts that this test was designed for. So yes, we can keep it, but I don't think we need to. cc @anjali411

Reply:
How about we keep it for now (just in case) but add a TODO: over it that we should review later?